tutor/commands/config.py

```diff
@@ -145,10 +152,13 @@ def save(
     remove_vars: list[tuple[str, t.Any]],
     unset_vars: list[str],
     env_only: bool,
+    clean_env: bool,
 ) -> None:
     config = tutor_config.load_minimal(context.root)
     if interactive:
-        interactive_config.ask_questions(config)
+        interactive_config.ask_questions(config, context.root)
```

Comment:
When I tested this, `tutor dev clean -ci` didn't ask me if I wanted to remove my env. I think you need to pass the `clean_env` through here.